groups/sig-cluster-lifecycle/groups.yaml

```diff
@@ -0,0 +1,15 @@
+groups:
+  - email-id: sig-cluster-lifecycle-cluster-api-alerts@kubernetes.io
+    name: sig-cluster-lifecycle-cluster-api-alerts
+    description: |-
+
+    settings:
+      WhoCanPostMessage: "ANYONE_CAN_POST"
+      ReconcileMembers: "true"
+    owners:
+      - detiber@gmail.com
+      - jdetiberus@equinix.com
+      - jeewan@vmware.com
+      - sbueringer@gmail.com
```

Comment:
I'll merge to allow us to iterate, but we need to stop making people owners of groups unless there are specific reasons to do so.

It's the same reason we disallow GitHub team owners.  If people can make manual changes, they will, and then get confused when our automation reverts their changes.

Reply:
Where would you want to see such a policy documented?  I plan on using tests to enforce, but that's the friendliest way to find out.

Reply:
🤔 We can start with an example of how a mailing list should be created in https://github.com/kubernetes/k8s.io/tree/master/groups and also add a link to https://www.kubernetes.dev/docs/comms/mailing-lists/ (I just found out).

Reply:
Oh, I was just copying the community one that had some folk in OWNERS. Happy to move everyone to members